quipucords/scanner/network/inspect.py

```diff
@@ -257,8 +262,8 @@ def _inspect_scan(self, connected, roles=DEFAULT_ROLES,
         inventory_file = write_inventory(inventory)
 
         error_msg = ''
-        log_message = 'START PROCESSING GROUPS with %d forks' \
-            ' and extra_vars=%s' % (forks, extra_vars)
+        log_message = 'START PROCESSING GROUPS with use_paramiko: %d, \
```

Comment:
Let's change this log to say with `openssh` or `paramiko` instead of use_paramiko.